content/en/docs/aws/pipeline.md

```diff
@@ -4,6 +4,29 @@ description = "Customize Kubeflow Pipelines to use AWS Services"
 weight = 90
 +++
 
+## Multi-user Kubeflow Pipelines
+For multi-user kubeflow pipelines, we provide a work-around when user want to connect to pipeline from kubeflow notebook:
+
+First, fetch public endpoint from command `kubectl get ingress -n istio-system` with prefix `https://`, then replace it with below `publicEndpoint`:
```

Comment:
This part is not ok. User need to authenticate the service with cookie.. I will hold this part and update it later. If you want to move the change forward. You can remove  this change on this file.

Reply:
Removed KFP part.